content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -116,10 +116,12 @@ The Kubelet populates this with `runtime.GOARCH` as defined by Go. This can be h
 
 Example: `kubernetes.io/os: "linux"`
 
-Used on: Node
+Used on: Node, Pod
 
 The Kubelet populates this with `runtime.GOOS` as defined by Go. This can be handy if you are mixing operating systems in your cluster (for example: mixing Linux and Windows nodes).
 
+When the `kubernetes.io/os` label in Pod labels does not match the node label, the kubelet will not admit the pod. However, this is not taken into account by kube scheduler.
```

Comment:
We might also want to link to https://kubernetes.io/docs/concepts/workloads/pods/#pod-os so that readers are aware that this label is one of two relevant mechanisms.

Reply:
I am unsure if this is the recommended approach.

> We might also want to link to https://kubernetes.io/docs/concepts/workloads/pods/#pod-os so that readers are aware that this label is one of two relevant mechanisms.

The Pod OS field seems an alternative way for the pod os label.